### DISCUSSION TITLE
Devices offline when Availability is on

### DISCUSSION THREAD

**Discussion by dsduarte** [Koenkk/zigbee2mqtt#11784] [Devices]
Hello guys...
I don't know if its an issue or not but a few days ago I turned on Availability and a few days later some devices started to fail, showing "offline" on Availability status.
When "offline" those devices doen't respond ping and map update fails.
If I turn Availability off and restart Z2M, those devices work fine.
I have noticed that this fails occur only on Tuya TS0002_switch_module and TS0012_switch_module.
First time I noticed that, I repaired  those devices and they were working just for a few days and then started to show off line again.

Thanks!

**Comment by jyavenard** [comment]
seeing the same thing. any updates?

**Comment by francisp2** [comment]
There is this warning in the docs about availability:

<img width="830" height="139" alt="afbeelding" src="https://github.com/user-attachments/assets/e3dc68e7-98f2-4717-8eac-b8f27e370435" />

**Comment by jyavenard** [comment]
In my case I'm using a CC2652R based adapter (Electrolama zig-a-zig-ah! (zzh!))
